src/docs/introduction/start-customizing/index.md

```diff
@@ -0,0 +1,149 @@
+Sharetribe-hosted client app, you can use the Sharetribe Web Template as
+your starting point. Sharetribe Web Template is an open-source version
+of the code that powers your Sharetribe-hosted marketplace.
+
+- [Read more about the Sharetribe Web Template](/introduction/introducing-template/)
+
+The template uses the TODO SDK to connect to the Marketplace API and
+Asset Delivery API. The SDK also handles authentication towards
+Marketplace API, and shares the user's login between the browser and the
+server.
+
+- [Read more about the TODO SDK](https://sharetribe.github.io/flex-sdk-js/)
+
+To get started with the template, you need to clone the Sharetribe Web
+Template repository and set it up with your marketplace client id and
+secret. The template uses your no-code configurations, fetched with the
+SDK, to render a marketplace that looks similar to your
+Sharetribe-hosted marketplace. You can then continue to custom develop
+features in your cloned codebase.
+
+- [Read more about getting started with the Sharetribe Web Template](/introduction/introducing-template/)
+
+The Sharetribe Web Template repository contains the starting point for
+your development work. To maintain version control of your own
+development work, you need to create your own remote repository for your
+cloned version of the template.
+
+The tutorial guides you through some common features that you can modify
+in the Sharetribe Web Template to custom develop your marketplace. We
+recommend that you check it through for an overview on the different
+customizations that are possible with the Sharetribe Developer Platform.
+
+- [Read the tutorial and learn about setting up a remote repository for your cloned codebase](/tutorial/introduction/)
+
+To share your changes online, you will need to deploy the template. Note
+that to deploy the template, for any marketplace environment (Dev, Test,
+or Live), your deployment service needs to be able to run a NodeJS
+server.
+
+- [Read more about template deployment](/template/how-to-deploy-template-to-production/)
+
+## Custom developing with your own client application
+
+You can also create a fully custom client application for your
+marketplace.
+
+We currently do not have a tutorial focusing on non-Sharetribe Web
+Template development. However, we recommend that you check through the
+template tutorial for an overview on what customizations are possible
+with the Sharetribe Developer Platform.
+
+- [Read the tutorial](/tutorial/introduction/)
+
+When deciding on the stack for your custom client application, it is
+good to note that to fully use the features of the Sharetribe Developer
+Platform, you need some kind of a server environment. Features requiring
+a server include:
+
+- using SSO and identity providers
+- Integration API – it provides full access to your marketplace, so it
+  must never be used from a browser environment
+- privileged transitions, including custom pricing
+- logging in as a user from Console
```

Comment:
What do you think about expanding on why these features require a server? Or would that be irrelevant in the context of this article

Reply:
My thinking here is that if someone wants to have SSO and needs to understand why a server is necessary, they would then read the article linked below. I wouldn't go into it too deep here, since it's more an overview article, but I'll see if I can come up with a few more words about needing a server